internal/sbi/consumer/nf_management.go

```diff
@@ -104,6 +109,11 @@ func SendDeregisterNFInstance() (*models.ProblemDetails, error) {
 	configuration.SetBasePath(nssfSelf.NrfUri)
 	client := Nnrf_NFManagement.NewAPIClient(configuration)
 
+	ctx, _, err = nssf_context.GetSelf().GetTokenCtx("nnrf-nfm", "NRF")
+	if err != nil {
+		return nil, errors.Errorf("SendDeregisterNFInstance error: %+v", err)
+	}
```

Comment:
Duplicate.

Reply:
Duplicate of [this part](https://github.com/free5gc/nssf/pull/18/files/cd88124876bd5858e7a90f49d5bb85b023f77b0e#diff-d9dca5ce560f622c1c5bca178498832cb33de74b7c5fe672e0bfe274fc00a731R46-R50)?